### DISCUSSION TITLE
[9.0.1] getDatabase references getApp that no longer exists

### DISCUSSION THREAD

**Discussion by anand-g** [firebase/firebase-js-sdk#5450] [Q&A]
Errors and after upgrading from 9.0.0-beta.8 to 9.0.1. 
I am new and could use some help - assume these are not bugs with the SDK.

Issue #1
With getApp no longer part of firebase/app, see this error.
```
ReferenceError: getApp is not defined
    at Module.getDatabase (/node_modules/@firebase/database/dist/index.esm2017.js?v=8bf0d814:13661:21)
```

Issue #2
getAuth() results in reference to window
```
ReferenceError: window is not defined
    at new BrowserLocalPersistence 
    at _getInstance 
    at _initializeAuthInstance (.../svelte-test/.svelte-kit/output/server/app.js:14361:80)
    at initializeAuth (.../.svelte-kit/output/server/app.js:14356:25)
    at getAuth (.../svelte-test/.svelte-kit/output/server/app.js:18391:10)
 ```

**Comment by Feiyang1** [comment]
For issue 1, it seems you may have a corrupted `node_modules`, can you remove it and do a fresh install?

For issue 2, it's a problem with Auth SDK in SSR. Can you update to 9.0.2 and follow the workaround [here](https://github.com/firebase/firebase-js-sdk/issues/5475#issuecomment-917616374)? We are planning to make SSR easier in the future, so you won't need the workaround (hopefully soon).
